low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -0,0 +1,260 @@
+#!/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+######################################################################
+# This script serves as an entry point for various operations for SSPL
+# as needed by provisioner.
+######################################################################
+
+import sys
+import errno
+import argparse
+import inspect
+import traceback
+import os
+import syslog
+import subprocess
+# Add the top level directories
+sys.path.insert(0, '/opt/seagate/cortx/sspl/low-level')
+
+class Cmd:
+    """Setup Command"""
+
+    def __init__(self, args: dict):
+        self._args = args.args
+        self._script_dir = os.path.dirname(os.path.abspath(__file__))
+
+    @property
+    def args(self) -> str:
+        return self._args
+
+    @staticmethod
+    def usage(prog: str):
+        """Print usage instructions"""
+
+        sys.stderr.write(
+            f"{prog} [post_install [-p <LDR_R1>]|init [-dp] [-r <vm>]\n|config [-f] [-r <vm>]|test [self|sanity]|reset [hard -p <LDR_R1>|soft]]\n"
```

Comment:
add remaining options too, which exists today already, like for manifest & support bundle